modules/Todos.test.js

```diff
@@ -21,4 +21,16 @@ describe('todos', () => {
       { description: 'church', completed: false, index: 0 },
     ]);
   });
+  test('check the checked items', () => {
+    const todo = new Todos();
+    todo.MarkCompleted([], 'book', true);
+    expect(JSON.parse(localStorage.getItem('todos'))).toEqual([
+      { description: 'book', completed: true, index: 0 },
+    ]);
+  });
+  test('Clear the checked items', () => {
+    const todo = new Todos();
+    todo.clearAllCompleted(2, true);
+    expect(JSON.parse(localStorage.getItem('todos'))).toEqual([{ description: 'John', completed: false, index: 0 }, { description: 'Mike', completed: false, index: 1 }]);
```

Comment:
- [ ] according to the requirements the function for editing the task description should be tested as well.